package.json

```diff
@@ -8,7 +8,12 @@
     "types": "type-declarations",
     "scripts": {
         "start": "npm run build && cd examples && npm run serve-example",
+        "start:watch": "npm-run-all --parallel watch \"serve-example:local\"",
+        "serve-example:local": "cd examples && npm run serve-example",
         "install-examples": "npm install && cd examples && npm install",
+        "watch": "npm-run-all --parallel watch:babel watch:css",
+        "watch:babel": "cross-env TS_NODE_PROJECT=tsconfig.es.json BABEL_ENV=es babel src --out-dir es --extensions .ts,.tsx,.js,.jsx --ignore 'src/test' --watch",
+        "watch:css": "postcss src/viewport/index.css --output style/style.css --watch",
```

Comment:
Adding all these things makes me nervous about complexity of build.  I wonder if the ones that match could be put close to each other, like `watch:css` next to `processCSS` and maybe rename processCSS to `build:css` or something more consistent.

Reply:
or even css:build and css:watch !  just riffing...